### ISSUE TITLE
Add new steady-state solver

### ISSUE THREAD

**Issue by kirichoi** [sys-bio/roadrunner#433]
New steady-state solver will attempt to brute-force a solution as a last resort.

**Comment by kirichoi** [comment]
Instead of adding a solver, I instead created a function called `steadyStateApproximate()`. This method will not be considered as a type of solver.

**Comment by kirichoi** [comment]
Right now, the default parameter works well. Matthias' model will results in 
```
[  5.31509103e-02,   5.87576642e-03,   7.26202218e-05,
   1.01952874e-01,   9.07635423e-05,   1.36254761e-02,
   6.48658684e-03,   4.48379152e-03,   4.83407435e-04,
   2.98136354e-01,   1.28959703e-01,   3.09631664e+00,
   1.00000000e-01,   3.00064508e-01,   9.62505934e-04,
   1.80209546e-06,   4.34275281e-04,   5.81342957e-02,
   2.32104689e+01,   1.17651456e-02,   1.49058056e-01,
   6.51673787e-05,   1.00000000e-02,   1.60714669e-01,
   2.90000000e-01,   4.50325629e-01,   8.99354915e-02]
```
which I think is close to the actual steady state.

**Comment by kirichoi** [comment]
Fixed with commit 3f212e83d86ab321b2db27d55e0e6e33e4ae5d8c
